com.unity.render-pipelines.core/Runtime/Textures/PowerOfTwoTextureAtlas.cs

```diff
@@ -1,12 +1,9 @@
-using System.Collections;
 using System.Collections.Generic;
-using UnityEngine;
-using UnityEngine.Rendering;
 using UnityEngine.Experimental.Rendering;
 
-namespace UnityEngine.Rendering.HighDefinition
+namespace UnityEngine.Rendering
 {
-    class PowerOfTwoTextureAtlas : Texture2DAtlas
+    public class PowerOfTwoTextureAtlas : Texture2DAtlas
```

Comment:
note: you will need to add documentation for it if we switch to public

Reply:
done.